client/i18n/locales/english/intro.json

```
@@ -1716,7 +1716,7 @@
       },
       "lecture-html-fundamentals": {
         "title": "HTML Fundamentals",
-        "intro": ["Learn about HTML fundamentals in these lecture videos."]
+        "intro": ["In these lecture videos, you will learn about HTML fundamentals like the <code>id</code> and <code>class</code> attributes, as well as the <code>div</code> and <code>span</code> elements, HTML entities, and more."]
```

```suggestion
        "intro": [
          "In these lecture videos, you will learn about HTML fundamentals like the <code>id</code> and <code>class</code> attributes, as well as the <code>div</code> and <code>span</code> elements, HTML entities, and more."
        ]
```
To make prettier happy.